packages/itwin/breakdown-trees/src/BreakdownTrees.ts

```diff
@@ -8,23 +8,27 @@
 import { UiError } from "@itwin/appui-abstract";
 import { getClassName } from "@itwin/appui-abstract";
 import type { Localization } from "@itwin/core-common";
-import type { LocalizationOptions } from "@itwin/core-i18n";
+import { IModelApp } from "@itwin/core-frontend";
+import { ITwinLocalization, LocalizationOptions } from "@itwin/core-i18n";
 
 /**
  * Entry point for static initialization required by various
  * components used in the package.
  * @public
  */
 export class BreakdownTrees {
-
+  private static _initialized?: boolean;
   private static _i18n?: Localization;
 
   /**
    * Called by IModelApp to initialize the BreakdownTrees
    * @param localization The internationalization service created by the IModelApp.
    */
-  public static async initialize(localization: Localization): Promise<void> {
-    BreakdownTrees._i18n = localization;
+  public static async initialize(localization?: Localization): Promise<void> {
+    if (this._initialized) return;
```

Comment:
Please see point no. 7 under [Make judicious use of vertical screen space section](https://github.com/iTwin/itwinjs-core/blob/master/docs/learning/guidelines/typescript-coding-guidelines.md#make-judicious-use-of-vertical-screen-space)

Reply:
changed